// terminal_service.proto
// Define the terminal service. This service is used to connect to a terminal
// that displays the meteorological data.

syntax = "proto3";

package terminal;

import "google/protobuf/empty.proto";
import "google/protobuf/timestamp.proto";

// TerminalService is the service that is used to connect to a terminal
service TerminalService {

  // Send processed data to the terminal
  rpc SendResults (Results) returns (google.protobuf.Empty);
}

message Results {
  float wellness_data = 1;
  google.protobuf.Timestamp wellness_timestamp = 2;
  float pollution_data = 3;
  google.protobuf.Timestamp pollution_timestamp = 4;
}
